index.html

```diff
@@ -2599,7 +2615,8 @@ <h2 id="docconformance">
                 <code>role=<a href="#index-aria-cell">cell</a></code> if a
                 descendant of a `table` element.
               </p>
-              <!--<p><code>role=<a href="#index-aria-cell">gridcell</a></code> if a descendant of a <code>table</code> elementwith <code>role=grid</code></p>-->
+              <!--<p><code>role=<a href="#index-aria-cell">gridcell</a></code> if a descendant
+                of a <code>table</code> elementwith <code>role=grid</code></p>-->
```

Comment:
"elementwith" > "element with"

Reply:
going to leave this as is.  going to be deleted / revised by another PR anyway.